app/assets/v2/js/vue-components.js

```diff
@@ -527,23 +527,6 @@ Vue.component('project-card', {
     }
   },
   methods: {
-    markWinner: function($event, project) {
-      let vm = this;
-
-      const url = '/api/v0.1/hackathon_project/set_winner/';
-      const markWinner = fetchData(url, 'POST', {
-        project_id: project.pk,
-        winner: $event ? 1 : 0
-      }, {'X-CSRFToken': vm.csrf});
-
-      $.when(markWinner).then(response => {
-        if (response.message) {
-          alert(response.message);
-        }
-      }).catch(err => {
-        console.log(err);
-      });
-    },
```

Comment:
is this removal related?

Reply:
As I didn't saw a comment about it on the description I requested changes. Not sure if product team is aware of that changes also.